Gruntfile.js

```diff
@@ -360,7 +360,7 @@ module.exports = function(grunt) {
   /**
    * Create custom task aliases and combinations.
    */
-  grunt.registerTask('compile-cf', ['bower:cf', 'concat:cf-less', 'concat:topdocIcons']);
+  grunt.registerTask('compile-cf', ['bower:cf', 'concat:topdocIcons']);
```

Comment:
Removed the `concat:cf-less` task and updated the comment above to be JS specific.